tests/tck/features/match/Scan.feature

```diff
@@ -155,3 +155,17 @@ Feature: Match seek by scan
       LIMIT 3
       """
     Then a ExecutionError should be raised at runtime: Scan vertices must specify limit number.
+    When executing query:
+      """
+      MATCH ()-[e:is_teacher]-()
+      RETURN type(e) AS Type, e.start_year AS StartYear, e.end_year AS EndYear
+      LIMIT 3
+      """
+    Then a ExecutionError should be raised at runtime: Scan vertices must specify limit number.
```

Comment:
The error msg really confuses me. What's the correct way to write this sentence?

Reply:
```
MATCH ()-[e:is_teacher]->()
      RETURN type(e) AS Type, e.start_year AS StartYear, e.end_year AS EndYear
      LIMIT 3
```

Reply:
Don't allow the bidirection.